docs/src/cookbooks/hierarchy.md

```diff
@@ -135,7 +135,12 @@ class Top extends Module {
 ```
 ```scala mdoc:passthrough
 println("```")
-chisel3.stage.ChiselStage.elaborate(new Top)
+val chiselCircuit = (new chisel3.stage.phases.Elaborate)
```

Comment:
why did this need to change?

Reply:
`circt.stage.ChiselStage` has no `elaborate` method. This avoids any usage of `chisel3.stage.ChiselStage` by doing this manually.

Reply:
This method should be added, but that would be a follow-up PR.